### PR TITLE
fixed all failing unit-test and system tests

### PR DESCRIPTION
**Pull request by dlyssenko**
there were a bunch of unit and system test failed (about 23 in total). This PR fixes all of them

### CODE REVIEW

File: pyeapi/utils.py [hunk 1]
```diff
@@ -243,6 +243,24 @@ class CliVariants:
     or with 2 or more sequences of cli (or a mix of list and str types), e.g.:
         ``CliVariants( ['new cli1', 'new cli2'], 'alt cli3', 'legacy cli4' )``
     """
+    @staticmethod
+    def expand( cmds ):
+        """cnds is a list of str and CliVariants, this method returns a list
```
Comment (mharista) [inline pyeapi/utils.py:248]: typo here? should this be cmds instead of cnds?

File: test/fixtures/dut.conf [hunk 2]
```diff
@@ -1,5 +1,4 @@
 [connection:veos01]
-host: 192.168.1.16
-username: eapi
-password: password
+host: roi401
```
Comment (mharista) [inline test/fixtures/dut.conf]: Is this the default value?

**Review by mharista — CHANGES_REQUESTED** [review]

File: pyeapi/utils.py (hunk 1 above)
Comment (dlyssenko) [reply pyeapi/utils.py:248]: yes, sharp eye, will fix this typo.

File: pyeapi/utils.py [hunk 3]
```diff
@@ -243,6 +243,24 @@ class CliVariants:
...
         ``CliVariants( ['new cli1', 'new cli2'], 'alt cli3', 'legacy cli4' )``
     """
+    @staticmethod
+    def expand( cmds ):
+        """cmds is a list of str and CliVariants, this method returns a list
+        of all full variant combinations present in cmds, e.g.:
+            expand( 'x', CliVariants( 'a', 'b'), 'y' )
+            will return: [ ['x', 'a', 'y'], ['x', 'b', 'y'] ]
+        """
+        assert isinstance(cmds, list), 'argument cmnds must be list type'
```
Comment (mharista) [inline pyeapi/utils.py:253]: uh oh another typo of cmds.

File: test/system/test_api_interfaces.py [hunk 4]
```diff
@@ -635,15 +653,16 @@ def test_update_vlan(self):
...
+            self.may_contain('vxlan vlan add 10 vni 10', dut, last=True)
 
     def test_remove_vlan(self):
         for dut in self.duts:
             dut.config(['no interface Vxlan1', 'interface Vxlan1'])
             api = dut.api('interfaces')
             instance = api.remove_vlan('Vxlan1', '10')
             self.assertTrue(instance)
-            self.notcontains('vxlan vlan remove 10 vni 10', dut)
+            self.notcontains('vxlan vlan remove 10 vni 10 $', dut)
```
Comment (mharista) [inline test/system/test_api_interfaces.py:665]: In a prior command usage the $ was part of a new format. Does an old vs new format not apply here?

**Review by mharista — CHANGES_REQUESTED** [review]
One more typo and one question.

File: test/system/test_api_interfaces.py (hunk 4 above)
Comment (dlyssenko) [reply test/system/test_api_interfaces.py:665]: yes, this is a test cli sequence, which I updated with the newer syntax, however to handle it properly the `remove_vlan ()` method was updated to handle both syntaxes - the newer and the deprecated one.

**Review by mharista — APPROVED** [review]

**Pull request merged**